### PR TITLE
Full audit: View basic graphs of activity

### PR DESCRIPTION
**Pull request by DonNateR**

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-0170afa9e6/how-to/)

**Comment by kmsholcomb** [comment]
Hi @DonNateR. Edits attached. Not much markup, but I have a suggestion for perhaps streamlining the graphs articles that exist for Rackspace Intelligence. There are three articles having to do with graphs (or that have "graphs" in their titles): 

- https://support.rackspace.com/how-to/viewing-basic-graphs-of-activity-in-rackspace-intelligence/
- https://support.rackspace.com/how-to/creating-custom-graphs-of-activity-in-rackspace-intelligence/
- https://support.rackspace.com/how-to/viewing-and-creating-graphs-of-activity-in-rackspace-intelligence-for-dedicated-accounts/

Now, that last article has "dedicated" in the title, but the information presented there does not appear to be any different than the information provided in the other two articles. In other words, there appears to be no difference in the way that Rackspace Intelligence handles graphs for cloud and dedicated accounts. If you want to get rid of duplicate information, you could consider combining all of the articles into one: use the "dedicated" one as the basis, and then just add in the details from the other two. And then, of course, delete "for dedicated accounts" from the title, and update all necessary links. That's probably more work than is worth doing, but if one of these articles ever needs true updating, you'll have to remember to update the other one. OK. Let me know if you have any questions about the edits. Thanks!

[View basic graphs of activity in Rackspace Intelligence_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1200930/View.basic.graphs.of.activity.in.Rackspace.Intelligence_editsKH.docx)
[View basic graphs of activity in Rackspace Intelligence_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1200929/View.basic.graphs.of.activity.in.Rackspace.Intelligence_editsKH.pdf)

**Comment by DonNateR** [comment]
@kmsholcomb: Already aware of the situation, we are sprucing up the articles we are keeping and retiring the about "Viewing and Creating". I have technical updates for the "Create" article, so that will be the next one I send you!

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-574bf6a777/how-to/)

File: content/rackspace-intelligence/view-basic-graphs-of-activity-in-rackspace-intelligence.md [hunk 1]
```diff
@@ -67,14 +62,9 @@ last time stamp.
...
 
-You can adjust the way your graphs are displayed on the page by
-selecting the appropriate display icon on the top right corner of the
-page.
-
-You can choose from one of the two display options:
-
--   *grid layout* - displays two graphs side by side
--   *linear layout* - displays your graphs in linear sequence
+You can adjust the way your graphs are displayed on the page by selecting the one of
```
Comment (stephamon) [inline content/rackspace-intelligence/view-basic-graphs-of-activity-in-rackspace-intelligence.md]: "selecting the one of the display" should be "selecting one of the display"

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c54618db4f/how-to/).

**Comment by stephamon** [comment]
👍

**Pull request merged**